plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -42,6 +43,7 @@ func Register(plugins *admission.Plugins) {
 			// load the configuration provided (if any)
 			configuration, err := LoadConfiguration(config)
 			if err != nil {
+				klog.Infof("Error while loading default config %v", err)
```

Comment:
this looks extraneous.  Why isn't it normally reported?

Reply:
Ya, it wasn't showing up in my log when testing locally. I did not debug it though. I will revert this for now and come back to it after debugging.